README.md

```diff
@@ -0,0 +1,53 @@
+# restbase-mod-table-sqlite [![Build Status](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite.svg?branch=master)](https://travis-ci.org/wikimedia/restbase-mod-table-sqlite)
+
+An SQLite3 back-end module for [RESTBase](wikimedia/restbase) conforming to the
+[RESTBase storage specification](wikimedia/restbase-mod-table-spec).
```

Comment:
This link is broken, it opens this URI: https://github.com/wikimedia/restbase-mod-table-sqlite/blob/readme/wikimedia/restbase-mod-table-spec